source/change-streams/tests/README.rst

```diff
@@ -160,6 +160,8 @@ The following tests have not yet been automated, but MUST still be tested
 #. Ensure that a cursor returned from an aggregate command with a cursor id and an initial empty batch is not closed on the driver side.
 #. The ``killCursors`` command sent during the "Resume Process" must not be allowed to throw an exception.
 #. ``$changeStream`` stage for ``ChangeStream`` against a server ``>=4.0`` and ``<4.0.7`` that has not received any results yet MUST include a ``startAtOperationTime`` option when resuming a changestream.
+#. ``$changeStream`` stage for ``ChangeStream`` started with ``startAfter`` against a server ``>=4.1.1`` that has not received any results yet MUST include a ``startAfter`` option and MUST NOT include a ``resumeAfter`` option when resuming a changestream.
+#. ``$changeStream`` stage for ``ChangeStream`` started with ``startAfter`` against a server ``>=4.1.1`` that has received at least one result MUST include a ``resumeAfter`` option and MUST NOT include a ``startAfter`` option when resuming a changestream.
```

Comment:
This looks like it's going to shift all of the numbers for later prose tests. I hope no drivers refer to them by number in their test suites.

Perhaps it'd be best not to use numbered bullets for these at all? If so, could we do that in a separate commit/PR?

Reply:
Yeah I agree here. I've recently started referring to prose tests via their number in the spec. I think it much easier to track which test is which.

Reply:
Sounds like @jmikola is advocating for getting rid of numbered tests, whereas @ShaneHarvey is in favor of keeping the numbers. Can you please clarify which course of action I should take?

Reply:
Oh sorry I misread Jeremy's comment. I am pro-numbers and I would like for new tests to be added at the end of the list. I think it's easier for test implementors to refer to tests via the number.

Reply:
I was advocating for removing the number bullets because tests may be removed (see: [SPEC-1360](https://jira.mongodb.org/browse/SPEC-1360)).

Reply:
For the time being, I will add the new tests to the end of the list. @jmikola I suppose we can pursue the question of whether to number the tests or not in a new JIRA ticket?

Reply:
Indeed. There's no need to do it here. I was just leaving the option open.

Reply:
https://jira.mongodb.org/browse/SPEC-1364